src/pages/index.js

```diff
@@ -15,28 +15,26 @@ const featureRows = [
       imageUrlDark: 'img/community_dark.svg',
       link: 'docs/chain/collator',
       description: (
-        <>
-          Become a collator in for our Peregrine Testnet and later help
-          decentralizing the KILT Blockchain.
-        </>
+        <>Become a collator and help decentralizing the KILT Blockchain.</>
       ),
     },
     {
       title: 'Workshop',
       imageUrl: 'img/expert_light.svg',
       imageUrlDark: 'img/expert_dark.svg',
       link: 'docs/sdk/workshop/welcome',
-      description: (
-        <>Explore the most fundamental features of the KILTprotocol.</>
-      ),
+      description: <>Claim, Attest and verify in our SDK workshop.</>,
     },
     {
-      title: 'Whitepaper',
+      title: 'Whitepaper 2020',
       imageUrl: 'img/whitepaper_light.svg',
       imageUrlDark: 'img/whitepaper_dark.svg',
       link: 'https://www.kilt.io/wp-content/uploads/2020/01/KILT-White-Paper-v2020-Jan-15.pdf',
       description: (
-        <>Read up on the theoretical concepts behind KILT Protocol.</>
+        <>
+          Read the original whitepaper. KILT evolved a lot since then, but the ideas and values are still part of our
+          identity even though the technical details are outdated.
```

Comment:
1 . Can we enforce new lines here, probably not? 
2. I  use a synonym for `a lot` as this is kind of informal/non-academical.
3. Maybe use `some technical details`?
```suggestion
          Read the original whitepaper.
          However, KILT has evolved significantly since writing it.
          Even though some of the included technical details are outdated by now, the underlying concepts, ideas and values are still part of our identity.
```

Reply:
since it's `.js` my formatter enforce line length of 120.

Reply:
I removed "a lot". Maybe we shouldn't use these qualifying language. It's too bury what is meant and doesn't add a lot of info. When we say "evolved a lot" what does it mean? Nothing is the same any more!? Same for the technical details. When we say some our outdated, which are the up to date details?

Reply:
Yeah definitely. These words don't add any value and can be removed.